### PR TITLE
initial draft of working agreement

### PR DESCRIPTION
**Pull request by bospoort**
## Description

This is a draft of the working agreement. I tried to adapt it from a full-time team to our part-time situation. 

It raises a few questions: 
- how do we handle sprint ceremonies (demo, retro, planning, grooming)
- how do we handle sizing of user stories for sprint planning. 

There is a lot in the doc. I consider it mostly a north star. Please add comments, objections, and additions. 

## What type of PR is this? (check all applicable)

- [ ] Refactor
- [ ] Feature
- [ ] Bug Fix
- [ ] Optimization
- [X] Documentation Update

## Related Tickets & Documents

https://das-ph-inventory-tracker.atlassian.net/browse/PIT-79

- Related Issue #
- Closes #

### CODE REVIEW

**Review by DrewLovell — COMMENTED** [review]
Reviewed. Will need to edit section on sprints.

File: docs/working-agreement.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+- We work as one team towards a common goal and clear scope
+- We make sure everyone's voice is heard, listened to
+- We show all team members equal respect
+- We work as a team to have common expectations for technical delivery
+- We make sure to spread our expertise and skills in the team, so no single person is relied on for one skill
+- All times below are listed in PST
+
+## Communication
+
+- We communicate all information relevant to the team through the Project Slack channel
```
Comment (aitzeng) [inline docs/working-agreement.md:17]: Suggest to mention we house meeting notes on Miro and ticket/user stories on Jiro

File: docs/working-agreement.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+- We add all technical spikes, trade studies, and other technical documentation to the project repository through async design reviews in PRs.
+
+## Work-life Balance
+
+- We work in a volunteering org and respect this, especially when setting up recurring meetings.
```
Comment (aitzeng) [inline docs/working-agreement.md:22]: Team members contribute minimum of 4 hours per week

**Comment by pingchenchan** [comment]
It looks good to me.

File: docs/working-agreement.md (hunk 2 above)
Comment (bospoort) [reply docs/working-agreement.md:23]: I added the 4 hours per week. Thanks!

File: docs/working-agreement.md (hunk 1 above)
Comment (bospoort) [reply docs/working-agreement.md:17]: Added. Good catch!

**Review by aitzeng — APPROVED** [review]

**Pull request merged**